sklearn/bench.py

```diff
@@ -607,3 +613,36 @@ def import_fptype_getter():
     except:
         from daal4py.sklearn.utils import getFPType
     return getFPType
+
+
+def patch_sklearn():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--patch_sklearn', type=str, default='None',
+                        choices=('None', 'True', 'False'),
+                        help='True for patch, False for unpatch, "None" to leave as is.')
+    args, _ = parser.parse_known_args()
+
+    if args.patch_sklearn is not None and args.patch_sklearn != 'None':
+        from daal4py.sklearn import patch_sklearn, unpatch_sklearn
+        if args.patch_sklearn == "True":
```

Comment:
Why are "True", "False" string, not boolean?

Reply:
One more option is "None" - to leave patching state "as is". think it can be useful for back compatibility. Possible we should change it to boolean.

Reply:
What will be broken in this case?